src/nvim/terminal.c

```diff
@@ -313,6 +313,10 @@ void terminal_resize(Terminal *term, uint16_t width, uint16_t height)
     return;
   }
 
+  if (height == 0 || width == 0) {
+    return;
+  }
```

Comment:
I see. Just for the record, normally, I'd request that such changes be put in the main PR, but I do prefer to get bug fixes merged sooner than later, and it's only one commit.

However, I still have to ask why not check for this in the above loop? It doesn't seem like `height` or `width` should be set to zero under any reasonable circumstances. Could `curwidth` have been zero initially? Can `wp->w_width` ever be zero? Does subtracting `win_col_off()` cause it? I worry that this might hide a bug rather than fix it (even if only temporarily).

Ideally, the commit message should document an analysis of the problem that justifies the proposed fix.